agents/smurf_file_emulator/smurf_file_emulator.py

```diff
@@ -92,6 +220,71 @@ def get_data_frame(self, start, stop):
         return fr
 
 
+class DataStreamer:
+    """
+    Helper class for streaming G3 data
+    """
+
+    def __init__(self, stream_id, sample_rate, tune, timestreamdir,
+                 file_duration, frame_len):
+        self.frame_gen = G3FrameGenerator(stream_id, sample_rate, tune)
+        self.session_id = self.frame_gen.session_id
+        self.stream_id = stream_id
+        self.timestreamdir = timestreamdir
+        self.seq = 0
+        self.file_duration = file_duration
+        self.file_start = 0
+        self.writer = None
+        self.file_list = []
+        self.frame_len = frame_len
+
+    def get_g3_filename(self):
```

Comment:
This looks like it was moved from the `SmurfFileEmulator` class. The old copy looks like it's not used anymore and could be removed.